### ISSUE TITLE
Symptoms worsen in post-partum state

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9990]
**Preferred term label:**
Symptoms worsen in post-partum state

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Multiple sclerosis
Telogen effluvium
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000091
- label: Postpartum exacerbation
- parents: [Aggravated by (HP:0025285)](https://hpo.jax.org/app/browse/term/HP:0025285)
- pmid: 
- def: Applied to a sign or symptom that is present during pregnancy and becomes worse following birth.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9990
- synonyms: Symptoms worsen in post-partum state(NONE)
